docs/src/docs/utilities/pdf-utilities.adoc

```diff
@@ -10,6 +14,31 @@ String textFromPdf = PdfUtils.getStringFromPdf(String pdfFileLocation);
 
 [source,java]
 ----
-//reads text from a pdf given as input steam.
-String textFromPdf = PdfUtils.getStringFromPdf(InputStream stream);
+//reads text from only one page of a pdf given as input steam.
+String textFromPdf = PdfUtils.getStringFromPdf(InputStream stream, int pageIndex);
+----
+
+== Exporting images
+
```

Comment:
Please add a simple example how to store an image from a single page of a PDF and add a comment how the file name is